docs/how-to-install.md

```diff
@@ -6,145 +6,170 @@ license: "This software is licensed under the Apache License version 2."
 
 ## Install Apache Gravitino from scratch
 
+### Prerequisites
+
+If you have not installed Java, install it and configure the `JAVA_HOME` environment variable.
+After installation, you can run the `${JAVA_HOME}/bin/java -version` command to check the Java version.
```

Comment:
Perhaps just `java -version`?

Reply:
This is the command of the original doc. I did not change the command. If you are sure, we can update it. If not, leave it here to keep it as before.

Reply:
leaving it is fine